legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-logical-plan/src/main/java/org/finos/legend/engine/persistence/components/util/SinkCleanupAuditDatasetAbstract.java

```diff
@@ -42,6 +42,12 @@ default String auditDatasetName()
         return DEFAULT_SINK_CLEAN_UP_AUDIT_TABLE;
```

Comment:
Let's delete this class if unused, we can bring it back when we implement it.

Reply:
Removed!